manuscripts/Dockerfile

```diff
@@ -11,12 +11,12 @@ CMD ["R", "-e", "rmarkdown::render('manuscript.Rmd')"]
 ## Non-interactive use:
 ## 
 ##     docker run --name rnexml cboettig/rnexml
-##     docker cp rnexml:/host/manuscript.pdf .
+##     docker cp rnexml:/home/rstudio/rnexml/manuscript.pdf `pwd`
 
 ## Equivalent to:
 ##  
-##     docker run --name rnexml --volumes $(pwd):/host \
-##       --workdir /host --user docker rocker/ropensci \
+##     docker run --name rnexml --volumes $(pwd):/home/rstudio/rnexml \
+##       --workdir /home/rstudio/rnexml --user docker rocker/ropensci \
```

Comment:
Just for the sake of clarity here: 

rocker/ropensci includes RNeXML and all CRAN-released R packages from ropensci.  However, since the manuscript itself is not a vignette, it is not shipped with the R package.  cboettig/rnexml just adds the manuscript files to rocker/opensci, but it's not the only way to do this.  

Sorry about the `--user docker`, the default user was renamed `rstudio` and that should have been changed.  It doesn't matter in the `cboettig/rnexml` one because we already set the default user in the Dockerfile itself.  It doesn't really matter even if we use `rocker/ropensci`, except that because the default user there is `root`, the resulting files created (`manuscript.pdf`) would be owned by `root` (at least on Linux, not sure how boot2docker handles that).